webpack/containers/Application/config.js

```diff
@@ -2,18 +2,20 @@ import Repos from '../../scenes/RedHatRepositories';
 import Subscriptions from '../../scenes/Subscriptions';
 import UpstreamSubscriptions from '../../scenes/Subscriptions/UpstreamSubscriptions/index';
 import SubscriptionDetails from '../../scenes/Subscriptions/Details';
+import SetOrganization from '../../components/SelectOrg/SetOrganization';
+import WithOrganization from '../../components/WithOrganization/withOrganization';
 
 // eslint-disable-next-line import/prefer-default-export
 export const links = [
   {
     text: 'RH Repos',
     path: 'redhat_repositories',
-    component: Repos,
+    component: WithOrganization(Repos, '/redhat_repositories'),
```

Comment:
HoC pattern, it's simple to add `Select Organization` logic to another page, just wrap it with `WithOrganization`